packages/graphql-cohttp/graphql-cohttp.0.10.0/opam

```diff
@@ -15,7 +15,7 @@ depends: [
   "ocaml" {>= "4.03.0"}
   "dune"
   "graphql" {>= "0.9.0"}
-  "cohttp" {>= "2.0.0"}
+  "cohttp" {>= "2.0.0" & < "6.0.0~~"}
```

Comment:
I'm unclear why you need the `~~` suffix with the version number… What's the reason behind it?

Reply:
It's the "lowest possible", i.e. lower than 6.0.0~beta1